core/src/main/java/tech/tablesaw/table/TableSlice.java

```diff
@@ -165,30 +167,23 @@ public double reduce(String numberColumnName, NumericAggregateFunction function)
   }
 
   /**
-   * Returns a 0 based int iterator for use with, for example, get(). When it returns 0 for the
-   * first row, get will transform that to the 0th row in the selection, which may not be the 0th
-   * row in the underlying table.
+   * Iterate of a copy of the table.
    */
   @Override
-  public IntIterator iterator() {
+  public Iterator<Row> iterator() {
```

Comment:
Is it possible to move this to `Relation` so that it doesn't have to be duplicated between here and `Table`?

Reply:
It is possible, but TableSice will still have to override it, which is fine. I will make the change.